api/v1/jwt.go

```diff
@@ -96,6 +96,9 @@ func JWTMiddleware(server *APIV1Service, next echo.HandlerFunc, secret string) e
 			if util.HasPrefixes(path, "/api/v1/memo") && method == http.MethodGet {
 				return next(c)
 			}
+			if util.HasPrefixes(path, "/api/v1/memo") && strings.HasSuffix(path, "/comment") && method == http.MethodPost {
```

Comment:
As I fixed the path value in #2014, we can use the route name to check if is matched.

```suggestion
			if path == "/api/memo/:memoId/comment" && method == http.MethodPost {
```